app/src/pages/inside/testCaseLibraryPage/addTestCasesToTestPlanModal/addTestCasesToTestPlanModal.tsx

```diff
@@ -0,0 +1,96 @@
+import { reduxForm } from 'redux-form';
+import { messages } from './messages';
+import { COMMON_LOCALE_KEYS } from 'common/constants/localization';
+import { useIntl } from 'react-intl';
+import { URLS } from 'common/urls';
+import { useSelector } from 'react-redux';
+import { projectKeySelector } from 'controllers/project';
+import { useMemo } from 'react';
+import classNames from 'classnames/bind';
+import { Modal } from '@reportportal/ui-kit';
+import styles from './addTestCasesToTestPlanModal.module.scss';
+import { LoadingSubmitButton } from 'components/loadingSubmitButton';
+import { useAddTestCasesToTestPlan } from './useAddTestCasesToTestPlan';
+import { AsyncAutocomplete } from 'componentLibrary/autocompletes/asyncAutocomplete';
+
+const cx = classNames.bind(styles) as typeof classNames;
+
+export const ADD_TO_TEST_PLAN_MODAL_KEY = 'addToTestPlanModalKey';
+
+interface TestPlan {
+  id: number;
+  name: string;
+}
+
+export const AddTestCasesToTestPlanModal = reduxForm<
+  unknown,
+  { selectedTestCases: (number | string)[] }
+>({
+  form: 'add-to-test-plan-modal-form',
+})(({ handleSubmit, selectedTestCases }) => {
+  const { formatMessage } = useIntl();
+
+  const projectKey = useSelector(projectKeySelector);
+
+  const {
+    isAddTestCasesToTestPlanLoading,
+    setSelectedTestPlan,
+    selectedTestPlan,
+    addTestCasesToTestPlan,
+  } = useAddTestCasesToTestPlan({
+    selectedTestCases,
+  });
+  const makeTestPlansOptions = (response: { content: TestPlan[] }) => response.content;
+
+  const description = useMemo(() => {
+    return (
+      <p className={cx('description')}>
+        <span>{formatMessage(messages.descriptionStart)}</span>
+        <b className={cx('selected-test-cases')}>{selectedTestCases?.length || 0}</b>
```

Comment:
`selectedTestCases?.length` is evaluated three times. I think you can use `size(selectedTestCases)` from lodash. Let's store it to variable. 

```
const selectedTestCasesCount = size(selectedTestCases);
```


```suggestion
        <b className={cx('selected-test-cases')}>{selectedTestCases}</b>
```